src/react-components/ui-root.js

```diff
@@ -44,17 +45,24 @@ import { faUsers } from "@fortawesome/free-solid-svg-icons/faUsers";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faQuestion } from "@fortawesome/free-solid-svg-icons/faQuestion";
 import { faInfoCircle } from "@fortawesome/free-solid-svg-icons/faInfoCircle";
+import { faArrowLeft } from "@fortawesome/free-solid-svg-icons/faArrowLeft";
 
 addLocaleData([...en]);
 
-const ENTRY_STEPS = {
-  start: "start",
-  device: "device",
-  mic_grant: "mic_grant",
-  mic_granted: "mic_granted",
-  audio_setup: "audio_setup",
-  finished: "finished"
-};
+// This needs to be updated as we add modal routes.
+const MODAL_ROUTES = [
```

Comment:
Could alternatively define this as a state on the route or for bonus points make a `ModalRoute` component that does so, so you define this property as part of defining the route.